classes/local/step/flow_sql.php

```diff
@@ -73,13 +73,13 @@ public function execute($input = null) {
 
         // Construct the query.
         $variables = $this->get_variables();
-        $config = $variables->get('config');
-        $sql = $this->evaluate_expressions($config->sql);
+        $config = $variables->get_raw('config');
+        list($sql, $params) = $this->evaluate_expressions($config->sql);
```

Comment:
```suggestion
        [$sql, $params] = $this->evaluate_expressions($config->sql);
```
An alternative which is shorter, and used more in modern languages (e.g. JS)

Reply:
Ah yeah I forgot the deconstruction got cleaner a while back. neat